docs/examplenotebooks.rst

```diff
@@ -0,0 +1,28 @@
+Example notebooks
+=================
+
+Some basic examples
+-------------------
+
+.. toctree::
+    :maxdepth: 3
+    :titlesonly:
+    :glob:
+
+    notebooks/example_projectq.ipynb
+    notebooks/grover_algorithm/grover_algorithm_qi.ipynb
+    notebooks/qi-performance-test.ipynb
+
+Classifier examples
+-------------------
+
+.. toctree::
+    :maxdepth: 3
+    :titlesonly:
+    :glob:
+
+    notebooks/classifier_example/classification_example1_2_data_points.ipynb
+    notebooks/classifier_example/classification_example2_4_data_points.ipynb
+    notebooks/classifier_example/classification_example3_4_features.ipynb
```

Comment:
I think new style you mean using token authentication?

I checked all notebooks but they all are already first loading the token and when that fails (not everybody has a token saved locally, e.g. Travis) it tries to load credentials from environment variables. When also that fails a username, password is asked.

Or did I misinterpret your comment?

Reply:
The code clutters the examples. I see this at multiple places:
```
QI_EMAIL = os.getenv('QI_EMAIL')
QI_PASSWORD = os.getenv('QI_PASSWORD')
QI_URL = os.getenv('API_URL', 'https://api.quantum-inspire.com/')

def get_authentication():
    """ Gets the authentication for connecting to the Quantum Inspire API."""
    token = load_account()
    if token is not None:
        return get_token_authentication(token)
    else:
        if QI_EMAIL is None or QI_PASSWORD is None:
            print('Enter email')
            email = input()
            print('Enter password')
            password = getpass()
        else:
            email, password = QI_EMAIL, QI_PASSWORD
        return get_basic_authentication(email, password)

authentication = get_authentication()
qi = QuantumInspireAPI(QI_URL, authentication)
```
If there are use cases that require the fallback to email/password, then put this code into a single function so in the notebooks we write
```
from quantuminspire.tools import get_authentication
authentication = get_authentication()
qi = QuantumInspireAPI(QI_URL, authentication)

```

Reply:
I agree it is repeating code, but actually not in scope of this ticket. But in this case I will fix it

I don't add this generic method to the SDK but to the repo itself. When someone uses the SDK for its own programs he/she will not use this construction for authentication but immediately token authentication or so. It is only when you don't know the method the user has (as is the case in the q-i-example repo).

Reply:
@QFer I think the method should be added to the QI repo. These notebooks are intended as runnable examples on the QI knowledge base. With this construction the user will have to add another package (the `quantum-inspire-examples`). 

The `quantum-inspire-examples` is not on pypi right? That means installing the package would require users to get a git repo and install, which is not the way we want to interact with new users.

As a side note: dashes in the name for the package is not ideal. We should replace them with `_` or choose a different name.

Reply:
@peendebak Didn't think of that, nice catch. I will move it to QI (created SDS-259 for it) but this creates an dependency between this new package and the next release of quantuminspire SDK. So before we release the examples, I have to release SDK again.

I know using dashes in module names is not done but in package names we do it all the time (control-software, api-router). That's actually why I choose for dashes instead of underscores. Why do you think this is a problem? Also on [example-pypi-package](https://pypi.org/project/example-pypi-package/) it is allowed and described as valid.